docs/website/versioned_docs/version-3.0.0/command-reference/delete.md

```diff
@@ -1,40 +1,39 @@
 ---
-title: odo delete
+title: odo delete component
 sidebar_position: 4
 ---
 
-`odo delete` command is useful for deleting resources that are managed by odo.
+`odo delete component` command is useful for deleting resources that are managed by odo. It deletes the component and it's related innerloop and outerloop resources from the cluster.
 
-## Deleting a component
-
-To delete a _devfile_ component, you can execute `odo delete`.
+There are 2 ways to deleting a component:
+1. [Delete with access to Devfile](/#Delete with access to Devfile)
+2. [Delete without access to Devfile](/#Delete without access to Devfile)
```

Comment:
This doesn't render correctly (look at the screenshot at the bottom after the suggestion). Should you remove the `/`?
```suggestion
1. [Delete with access to Devfile](#Delete with access to Devfile)
2. [Delete without access to Devfile](#Delete without access to Devfile)
```
----
Screenshot:
![image](https://user-images.githubusercontent.com/713521/159653871-6c923a68-5d9f-4f93-ba9a-027857f9ae25.png)

Reply:
That still doesn't work. I had your suggestion earlier on, and it showed them the way it does in your screenshot.

Reply:
Maybe try (check both case-sensitive and case-insensitive):
```md
[Delete without access to Devfile](#Delete-without-access-to-Devfile)
```

Reply:
```suggestion
1. [Delete with access to Devfile](#delete-with-access-to-devfile)
2. [Delete without access to Devfile](#delete-without-access-to-devfile)
```